cypress/e2e/home.cy.ts

```diff
@@ -3,6 +3,6 @@ describe('Visits the home page', () => {
         cy.visit('/')
     })
     it('Displays the Hero Section', () => {
-        cy.get('.h-full ').contains('The open-source project that\'s helping Japan get vaccinated')
+        cy.get('h1').contains('Connecting people in Japan to the healthcare services they need, in the languages they need.')
```

Comment:
In cypress it's discouraged to make selectors too attached to CSS, which will change.

Reply:
Totes agree with this! To your point, we'll need to set up cypress testing standard practices once we have some